internals/generators/container/stateless.js.hbs

```diff
@@ -46,7 +55,7 @@ function {{ properCase name }}() {
 
 {{#if wantActionsAndReducer}}
 const mapStateToProps = createStructuredSelector({
-  {{name}}: makeSelect{{properCase name}}(),
+  {{ lowerCase name }}: makeSelect{{properCase name}}(),
```

Comment:
`createStructuredSelector` helper was here for a long time if my memory serves me right, it accepts selectors, not selector factories and `makeSelect..` is a selector factory. If it was not a factory we can write:

```
const structuredSelector = createStructuredSelector({
  x: mySelectorA,
  y: mySelectorB
})
```